code/modules/power/apc.dm

```diff
@@ -681,7 +681,7 @@
 		"chargingStatus" = charging,
 		"totalLoad" = lastused_total,
 		"coverLocked" = coverlocked,
-		"siliconUser" = user.has_unlimited_silicon_privilege,
+		"siliconUser" = user.has_unlimited_silicon_privilege || locate(/obj/machinery/computer/apc_control) in range(1, user),
```

Comment:
Through some testing this isn't very cooperative to pull off. This is snowflakey but I think it's the best solution I have. For context, here's what I tried:

```
var/obj/machinery/computer/apc_control/A = locate() in range(1, user)
if(A)
  data["siliconUser"] = A.owner == user
```

That was below the rest of the assignment.